### ISSUE TITLE
Error in Hungarian translation

### ISSUE THREAD

**Issue by balazsorban44** [date-fns/date-fns#1001]
Moving it to a new issue from #844 for clarification. @TwoDCube, @reyrodrigez did you guys work on the translations? If so, could you have a look at this?

I am using the HU 🇭🇺 locale, and unfortunately, it gives a grammatically wrong result:
```js
formatDistance(
    new Date('A date that is 2 days ago'),
    new Date(),
    {locale: hu, addSuffix: true}
)
// => 2 napja ezelőtt
// Correct: 2 napja or 2 nappal ezelőtt
```
I am using `date-fns@next`

**Comment by TwoDCube** [comment]
Hi! I've opened a PR for this. What do you think?
```
formatDistance(
    new Date('A date that is 2 days ago'),
    new Date(),
    {locale: hu, addSuffix: true}
)
// => 2 nappal ezelőtt

formatDistance(
    new Date('A date that is 2 days ago'),
    new Date(),
    {locale: hu, addSuffix: false}
)
// => 2 napja
```

**Comment by balazsorban44** [comment]
Looks like it would work! :ok_hand: waiting for merge. 🙂

**Comment by reyrodrigez** [comment]
thanks @TwoDCube, greatjob!

**Comment by kossnocorp** [comment]
This fix for the issue was released with `v2.0.0-alpha.26`. See: https://blog.date-fns.org/post/v200-alpha26-is-out-2rizgx9wk2g1t
